Cannot fetch manifests due to misconfigured channels in monitor

When https://github.com/mozilla/experimenter/pull/9280 landed, the channels for monitor were `nightly` and `developer`. However, as of https://github.com/mozilla/blurts-server/pull/3320, the channels are now `beta` and `release`. This causes the feature manifest update task to break for all platforms:

```
     Running `/home/circleci/experimenter/experimenter/experimenter/features/manifests/application-services/target/debug/nimbus-fml ../../../../monitor-web.fml.yaml experimenter --output ../../../../monitor-web.yaml --channel developer`
Error: Invalid Channel error: The channel `developer` is specified, but only ["beta", "release"] are supported for the file
make: *** [Makefile:94: convert_feature_manifests] Error 1
```

[failed pipeline](https://app.circleci.com/pipelines/github/mozilla/experimenter/27983/workflows/90600991-5981-451e-a6e4-a952c1f83f74)

--- a/Makefile
+++ b/Makefile
@@ -99,7 +99,7 @@ convert_feature_manifests:
 	git submodule init;\
 	git submodule update --recursive;\
 	cd components/support/nimbus-fml;\
-	cargo run "../../../../monitor-web.fml.yaml" experimenter --output "../../../../monitor-web.yaml" --channel developer;
+	cargo run generate-experimenter --channel beta "../../../../monitor-web.fml.yaml" "../../../../monitor-web.yaml";
 
 fetch_external_resources: jetstream_config feature_manifests convert_feature_manifests
 	echo "External Resources Fetched"

